### ISSUE TITLE
UX edits

### ISSUE THREAD

**Issue by Uzval-Mallepeddi** [uclibs/application_portfolio#127]
As per May's suggestions,

1) Change the app name to "UCL Application Portfolio" whither applicable.
2) Remove the "jiggling" animation (bounce).
3) Color contrast issues for footer on login page. Make "All Rights Reserved ® 2019" to black/white as per visibility.
4) All the columns displaying the info. should be left-justified.
5) Avoid gray background.
